bundle/compliance/cis_azure/rules/cis_7_2/rule.rego

```diff
@@ -0,0 +1,21 @@
+package compliance.cis_azure.rules.cis_7_2
+
+import data.compliance.lib.common
+import data.compliance.policy.azure.data_adapter
+
+finding = result {
+	# filter
+	data_adapter.is_vm
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(has_managed_disk),
+		{"Resource": data_adapter.resource},
+	)
+}
+
+default has_managed_disk := false
+
+has_managed_disk {
+	data_adapter.properties.storageProfile.osDisk.managedDisk.id != ""
```

Comment:
is it an empty string? or is it completely missing?

Reply:
I am just checking the id exists as a sanity check, it's the one field that should always be there.